test/functional/mempool_accept.py

```diff
@@ -281,13 +280,6 @@ def run_test(self):
             result_expected=[{'txid': tx.rehash(), 'allowed': False, 'reject-reason': 'dust'}],
             rawtxs=[tx.serialize().hex()],
         )
-        tx.deserialize(BytesIO(hex_str_to_bytes(raw_tx_reference)))
-        tx.vout[0].scriptPubKey = CScript([OP_RETURN, b'\xff'])
-        tx.vout = [tx.vout[0]] * 2
-        self.check_mempool_result(
-            result_expected=[{'txid': tx.rehash(), 'allowed': False, 'reject-reason': 'multi-op-return'}],
```

Comment:
I'd suggest not removing the test case, but rather simply adapting the expected result (to `'allowed': True` instead of False and removing the `'reject-reason'`).